test/mocks/config-backend-server.js

```diff
@@ -0,0 +1,168 @@
+const Koa = require('koa');
+const Router = require('@koa/router');
+const bodyParser = require('koa-bodyparser');
+const { transformationMocks, libraryMocks, rudderLibraryMocks } = require('./test-data/api-responses');
```

Comment:
Can we provide responses dynamically instead of having to add everything to a single file?